dashboard/test/factories/factories.rb

```diff
@@ -150,6 +150,58 @@
           end
         end
       end
+      trait :with_google_authentication_option do
+        after(:create) do |user|
+          create(:authentication_option,
+            user: user,
+            email: user.email,
+            hashed_email: user.hashed_email,
+            credential_type: 'google_oauth',
```

Comment:
`google_oauth` should actually be `google_oauth2`, which is a mistake I made when I wrote this code, and fix up in https://github.com/code-dot-org/code-dot-org/pull/23136/ - if we do need to duplicate these traits, it probably makes sense to fix this wherever they're duplicated.

Reply:
i'll update this in a cleanup PR too!